docs/conf.py

```diff
@@ -273,7 +273,7 @@
 # -- Options for intersphinx module --------------------------------------------
 
 intersphinx_mapping = {
-    'http://docs.python.org/': ('https://docs.python.org/3.6', None),
+    'http://docs.python.org/': ('https://docs.python.org/3.8', None),
```

Comment:
Maybe just refer to `docs.python.org/3/` which would open the most recent stable version (3.8.5 in this case)

Reply:
Good point. Thanks!